plugins/woocommerce/src/Internal/Admin/Marketplace.php

```diff
@@ -71,4 +78,59 @@ public function enqueue_scripts( $hook_suffix ) {
 		// Enqueue WordPress updates script to enable plugin and theme installs and updates.
 		wp_enqueue_script( 'updates' );
 	}
+
+	/**
+	 * Add a Woo Marketplace link to the plugin install action links.
+	 * @param array $tabs Plugins list tabs.
+	 * @return array
+	 */
+	public function add_woo_plugin_install_action_link( $tabs ) {
+		$tabs[self::MARKETPLACE_TAB_SLUG] = _x( 'Woo', 'Plugin Installer', 'wccom' );
+		return $tabs;
+	}
+
+	/**
+	 * Open the Woo tab when the user clicks on the Woo link in the plugin installer.
+	 */
+	public function maybe_open_woo_tab() {
+		if ( ! isset( $_GET['tab'] ) || self::MARKETPLACE_TAB_SLUG !== $_GET['tab'] ) {
+			return;
+		}
+
+		$woo_url = add_query_arg(
+			array(
+				'page' => 'wc-admin',
+				'path' => '/extensions',
+				'tab'  => 'extensions',
+				'ref'  => 'plugins',
+			),
+			admin_url( 'admin.php' )
+		);
+
+		wp_safe_redirect( $woo_url );
+		exit;
+	}
+
+	/**
+	 * Add styles to the plugin install page.
+	 */
+	public function add_plugins_page_styles() {
+		?>
+		<style>
+			.plugin-install-woo > a::after {
+				content: "";
+				display: inline-block;
+				background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' viewBox='0 0 16 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M8.33321 3H12.9999V7.66667H11.9999V4.70711L8.02009 8.68689L7.31299 7.97978L11.2928 4H8.33321V3Z' fill='%23646970'/%3E%3Cpath d='M6.33333 4.1665H4.33333C3.8731 4.1665 3.5 4.5396 3.5 4.99984V11.6665C3.5 12.1267 3.8731 12.4998 4.33333 12.4998H11C11.4602 12.4998 11.8333 12.1267 11.8333 11.6665V9.6665' stroke='%23646970'/%3E%3C/svg%3E%0A");				width: 24px;
+				width: 16px;
```

Comment:
Stray `width` rule here.

```suggestion
				background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' viewBox='0 0 16 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M8.33321 3H12.9999V7.66667H11.9999V4.70711L8.02009 8.68689L7.31299 7.97978L11.2928 4H8.33321V3Z' fill='%23646970'/%3E%3Cpath d='M6.33333 4.1665H4.33333C3.8731 4.1665 3.5 4.5396 3.5 4.99984V11.6665C3.5 12.1267 3.8731 12.4998 4.33333 12.4998H11C11.4602 12.4998 11.8333 12.1267 11.8333 11.6665V9.6665' stroke='%23646970'/%3E%3C/svg%3E%0A");
				width: 16px;
```